I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/combat/combat screen.rpy", line 49, in script call
    call combat
  File "game/combat/combat screen.rpy", line 113, in script
    pause ##gameplay here
  File "renpy/common/000statements.rpy", line 455, in execute_pause
    renpy.pause()
  File "game/combat/combat screen.rpy", line 117, in execute
    screen combat:
  File "game/combat/combat screen.rpy", line 117, in execute
    screen combat:
  File "game/combat/combat screen.rpy", line 142, in execute
    if aztimer > 0:
  File "game/combat/combat screen.rpy", line 142, in <module>
    if aztimer > 0:
NameError: name 'aztimer' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/combat/combat screen.rpy", line 49, in script call
    call combat
  File "game/combat/combat screen.rpy", line 113, in script
    pause ##gameplay here
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\ast.py", line 2212, in execute
    self.call("execute")
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\ast.py", line 2200, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\statements.py", line 281, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 455, in execute_pause
    renpy.pause()
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\exports.py", line 1533, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward, pause=delay)
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\display\core.py", line 3376, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs) # type: ignore
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\display\core.py", line 3803, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\display\screen.py", line 451, in visit_all
    callback(self)
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\display\core.py", line 3803, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\display\screen.py", line 462, in per_interact
    self.update()
  File "C:\Renpy\renpy-8.0.0-sdk\renpy\display\screen.py", line 653, in update
    self.screen.function(**self.scope)
  File "game/combat/combat screen.rpy", line 117, in execute
    screen combat:
  File "game/combat/combat screen.rpy", line 117, in execute
    screen combat:
  File "game/combat/combat screen.rpy", line 142, in execute
    if aztimer > 0:
  File "game/combat/combat screen.rpy", line 142, in <module>
    if aztimer > 0:
NameError: name 'aztimer' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.0.0.22062402
Erasers 1.0
Thu Jun 22 01:01:55 2023
